code/game/gamemodes/clock_cult/clock_structures.dm

```diff
@@ -220,14 +211,9 @@
 						L.adjust_fire_stacks(damage_per_tick)
 						L.IgniteMob()
 			else if(istype(target,/obj/mecha))
-				var/sending_message = FALSE
-				if(mech_damage_cycle > 1)
-					mech_damage_cycle = 0
-					sending_message = TRUE
-				else
-					mech_damage_cycle++
 				var/obj/mecha/M = target
-				M.take_directional_damage(damage_per_tick, "fire", get_dir(src, M), 0, 1, sending_message)
+				M.take_damage(damage_per_tick, BURN, "melee", 1, get_dir(src, M)) //does about half of standard damage to mechs * whatever their fire armor is
```

Comment:
the spam is dampened by the fact that mech occupant only see an attack message 33% of the time when the attack is 5 hitpoints or less.
The "fire" arg in take_directional_damage() corresponds to the BURN below it. Mech have suffered a nerf, they previously had specific damage absorption for brute and fire, but I had to use "melee" instead which means they use pretty much the same protection for both burn damage and brute damage. I'm thinking of fixing it by splitting the melee flags into melee_brute and melee_burn, but that's for another PR. So yeah it's kind of a nerf for the moment.